.github/workflows/release-images.yaml

```
@@ -111,6 +147,16 @@ jobs:
             IMAGE_NAMESPACE=${{ github.repository }} \
             PLATFORM=${{ matrix.platform }}/${{ matrix.arch }} \
             BUILDX_ACTION=--push
+          
+      - name: Sign container image
+        run: |
+          for image in \
+            ghcr.io/${{ github.repository }}/retina-operator:$TAG \
+            ghcr.io/${{ github.repository }}/retina-operator:$TAG-${{ matrix.platform }}-${{ matrix.arch }} \
```

```suggestion
            ghcr.io/${{ github.repository }}/retina-operator:$TAG-${{ matrix.platform }}-${{ matrix.arch }} \
```
same comment re: manifests